build/style.css

```diff
@@ -0,0 +1,80 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;900&display=swap");
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+}
+body {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  background-position: 20px 10px;
+  background-image: url(./header-shapes\ mobile.png);
+  background-size: cover;
+  background-position-y: 1px;
+  background-position-x: 30px;
```

Comment:
- [ ] I'd suggest you use `background-repeat: no-repeat;` so the background design does not repeat🤷
- Your design
![screenshot (26)](https://user-images.githubusercontent.com/44624138/148444339-2647ebd9-53fd-4cdf-8898-ba087afa281a.png)

Reply:
Thank you so much for those corrections, they are very helping me.